hieradata_aws/class/monitoring.yaml

```
@@ -1,5 +1,8 @@
 ---
 
+govuk_safe_to_reboot::can_reboot: 'careful'
+govuk_safe_to_reboot::reason: 'We lose alerting when this machine is down'
```

Would a more specific statement fit or would this be too long? Seems like missing those job status reports is the key thing that folks didn't realise here.

```suggestion
govuk_safe_to_reboot::reason: 'We lose monitoring and can miss passive alerts while this machine is down because jobs cannot report their status'
```

---

I think that's about the same as the original for me. It's not a big deal - it's certainly true that we lose alerting while the machine is down so let's go with that.